examples/versioning-test.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"github.com/emicklei/go-restful"
+	"net/http"
+	"reflect"
+)
+
+// simple struct for registering new entity accessor
+type keyvalue struct {
+	readCalled bool
+	writeCalled bool
+}
+
+// generic read function
+func (kv *keyvalue) Read(req *restful.Request, v interface{}) error {
+	kv.readCalled = true
+	return nil
+}
+
+// generic write function
+func (kv *keyvalue) Write(resp *restful.Response, status int, v interface{}) error {
+	t := reflect.TypeOf(v)
+	rv := reflect.ValueOf(v)
+	for ix := 0; ix < t.NumField(); ix++ {
+		sf := t.Field(ix)
+		io.WriteString(resp, sf.Name)
+		io.WriteString(resp, "=")
+		io.WriteString(resp, fmt.Sprintf("%v\n", rv.Field(ix).Interface()))
+	}
+	kv.writeCalled = true
+	return nil
+}
+
+func main() {
+	kv := new(keyvalue)
+
+	// create new entity accessor for application/test+v1 since it is not standard
+	restful.RegisterEntityAccessor("application/test+v1", kv)
```

Comment:
Do we want to send this content-type: application/test+v1 from client(browser)? 
or do we want to send something like : application/test;v=1